packages/grafana-ui/src/components/UsersIndicator/types.ts

```diff
@@ -0,0 +1,12 @@
+import { DateTimeInput } from '@grafana/data';
+
+export interface UserView {
+  user: {
+    /** User's name, containing first + last name */
+    name: string;
+    /** URL to the user's avatar */
+    avatarUrl?: string;
+  };
+  /** Datetime string when the user was last active */
+  lastActiveAt: DateTimeInput;
```

Comment:
Does this have to be ISO UTC?

Reply:
Can be anything that our `dateTime` function can convert.